contributing/docs-style.md

```diff
@@ -0,0 +1,366 @@
+
+  </div>
+  {% endhighlight %}
+
+  These code blocks are often used to learn and explore, so there are a few best practices to keep in mind:
+
+  * **Showcase key development tasks.** Reserve code examples for common implementation scenarios that are meaningful for users. Leave more lengthy and complicated examples for tutorials or walkthroughs.
+
+  * **Ensure the code is standalone.** Code examples should be self-contained and not have external dependencies (except for outlier cases such as examples on how to use specific connectors). Include all import statements without using wildcards, so that newcomers can understand and learn which packages are being used.
+
+  * **Avoid shortcuts.** For example, handle exceptions and cleanup as you would in real-world code.
+
+  * **Include comments, but don’t overdo it.** Provide an introduction describing the main functionality of the code and possible caveats that might not be obvious from reading it. Use comments to clarify implementation details and to describe the expected output.
+
+[Back to top](#top)
+
+## The Guiding Principles of ACORN
+
+This style guide has the overarching goal of setting the foundation for documentation that is **Accessible**, **Consistent**, **Objective**, **Rational** and **(I)Nclusive**.
+
+#### Accessible
+
+The Flink community is diverse and international, so you need to think wide and globally when writing documentation. Not everyone speaks English at a native level and the level of experience with Flink (and stream processing in general) ranges from absolute beginners to experienced advanced users. Ensure technical accuracy and linguistic clarity in the content you produce so that it can be understood by all users.
+
+#### Consistent
+
+Stick to the basic guidelines detailed in this style guide and use your own best judgment to uniformly spell, capitalize, hyphenate, bold and italicize text. Correct grammar, punctuation and spelling are desirable, but not a hard requirement — documentation contributions are open to any level of language proficiency.
+
+#### Objective
+
+Keep your sentences short and to the point. As a rule of thumb, if a sentence is shorter than 14 words, readers will likely understand 90 percent of its content. Sentences with more than 25 words are usually harder to understand and should be revised and split, when possible. Being concise and using well-known keywords also allows users to navigate to relevant documentation with little effort.
+
+#### Rational
+
+Be mindful that most users will scan through online content and only read around [28 percent of it](https://www.nngroup.com/articles/website-reading/). This underscores the importance of grouping related ideas together into a clear hierarchy of information and using focused, descriptive headings. Placing the most relevant information in the first two paragraphs of each section is a good practice that increases the “return of time invested” for the user.
+
+#### (I)Nclusive
```

Comment:
Why is the I in parentheses?
I expect this to be a double-meaning thing "Inclusive" and "Nclusive". But understanding this stops at "Nclusive".
I would propose to write "Inclusive" here, instead of this construct.

Reply:
It was a stretched attempt at an acronym. I'll rewrite this as "Guiding Principles" to make it clear.